packages/react-positioning/src/utils/hasAutoFocusFilter.ts

```diff
@@ -0,0 +1,31 @@
+//
+// Dev utils to detect if nodes have "autoFocus" props.
+//
+
+import { getReactFiberFromNode } from './getReactFiberFromNode';
+
+/**
+ * Detects if a passed HTML node has "autoFocus" prop on a React's fiber. Is needed as React handles autofocus behavior
+ * in React DOM and will not pass "autoFocus" to an actual HTML.
+ *
+ * @param {Node} node
```

Comment:
tip: js doc types are not needed in typescript and should be used solely in JS

Reply:
removed in [ef658f4](https://github.com/microsoft/fluentui/pull/21828/commits/ef658f4645a1043ee2e858cf706c7a63407f5b29)